chain/epoch-manager/src/proposals.rs

```diff
@@ -45,11 +45,13 @@ pub fn proposals_to_epoch_info(
     validator_kickout: HashMap<AccountId, ValidatorKickoutReason>,
     validator_reward: HashMap<AccountId, Balance>,
     minted_amount: Balance,
-    current_version: ProtocolVersion,
-    next_version: ProtocolVersion,
+    prev_prev_protocol_version: ProtocolVersion,
```

Comment:
This is a bit ambiguous - it could be protocol version from the prev prev epoch or protocol version - 2. I don't have a good suggestion unfortunately. Maybe prev_prev_epoch_protocol_version? Maybe add a comment?

Reply:
Makes sense.